client/gutenberg/extensions/map-block/load-script/index.js

```diff
@@ -0,0 +1,65 @@
+/** @format */
+/**
+ * A little module for loading a external script
+ *
+ * @format
+ */
+
+/**
+ * External dependencies
+ */
+import debugFactory from 'debug';
+const debug = debugFactory( 'lib/load-script' );
+
+/**
+ * Internal dependencies
+ */
+import { addScriptCallback, isLoading } from './callback-handler';
+import { createScriptElement, attachToHead } from './dom-operations';
+
+// NOTE: This exists for compatibility.
+export { removeScriptCallback } from './callback-handler';
+
+/**
+ * Module variables
+ */
+export const JQUERY_URL = 'https://s0.wp.com/wp-includes/js/jquery/jquery.js';
+
+//
+// loadScript and loadjQueryDependentScript
+//
+
+export function loadScript( url, callback ) {
+	// If this script is not currently being loaded, create a script element and attach to document head.
+	const shouldLoadScript = ! isLoading( url );
+
+	addScriptCallback( url, callback );
+
+	if ( shouldLoadScript ) {
+		attachToHead( createScriptElement( url ) );
+	}
+}
+
+export function loadjQueryDependentScript( url, callback ) {
```

Comment:
I'm curious why we need this - it doesn't seem to be used anywhere.

Reply:
Quick note about this, and all of `load-script`... This is a Calypso library found here: https://github.com/Automattic/wp-calypso/tree/master/client/lib/load-script. I want to directly import the original, but doing so leads to a breaking error that I filed here: https://github.com/Automattic/wp-calypso/issues/27821. For a short term fix I copied the library into the block directory, and hackily ditched the lines that were causing the error, which unblocked me. As soon as the issue is resolved, I'll remove the duplicated library and change the import. So the short answer here is, there's not much point making any changes to `load-script` since it's not long for this world :-)